src/main/features/claimant-response/views/confirmation.njk

```diff
@@ -48,6 +56,14 @@
         {% include "./confirmation/proposed-alternate-payment-plan.njk" %}
       {% elseif repaymentPlanOrigin === 'COURT' %}
         {% include "./confirmation/court-suggested-payment-plan.njk" %}
+      {% elseif claim.claimantResponse.type === 'acceptation' %}
```

Comment:
can you add the ClaimantResponse enum as a global variable in nunjuks so we don't have to evaluate string literals?